gradle/docker.gradle

```diff
@@ -0,0 +1,66 @@
+buildscript {
```

Comment:
what actually calls this file? Do you have to call it separately to run the tests or does calling the gradle test command handle it?

Reply:
The main file build.gradle  includes it